src/main/java/com/puppycrawl/tools/checkstyle/xpath/AttributeNode.java

```diff
@@ -182,6 +184,27 @@ public DetailAST getUnderlyingNode() {
         throw throwUnsupportedOperationException();
     }
 
+    /**
+     * Creates nodes for children. Attribute node has no children, so
+     * this method throws {@code UnsupportedOperationException}.
+     *
+     * @return never
```

Comment:
probably better to say "throw unsupported exception"

Reply:
done

Reply:
not done.

Reply:
Sorry for mistype. Fixed.